lib/memstore.js

```diff
@@ -59,18 +59,19 @@ MemoryCookieStore.prototype.findCookies = function(domain, path, cb) {
     };
 
   } else {
-    var paths = permutePath(path) || [path];
     pathMatcher = function matchRFC(domainIndex) {
-      paths.forEach(function(curPath) {
-        var pathIndex = domainIndex[curPath];
-        if (!pathIndex) {
-          return;
-        }
-        for (var key in pathIndex) {
-          results.push(pathIndex[key]);
-        }
-      });
-    };
+       //NOTE: we should use path-match algorithm from S5.1.4 here
+       //(see : https://github.com/ChromiumWebApps/chromium/blob/b3d3b4da8bb94c1b2e061600df106d590fda3620/net/cookies/canonical_cookie.cc#L299)
```

Comment:
Could you please explain here why we should use that path-match implementation?

Tactical: if this is a TODO, then maybe that goes as a comment on `pathMatch`? Or are you saying that `matchRFC` doesn't sync up with how Chromium does it?